ecdsa.c

```diff
@@ -829,6 +849,30 @@ void ecdsa_get_pubkeyhash(const uint8_t *pub_key, uint8_t *pubkeyhash)
 	MEMSET_BZERO(h, sizeof(h));
 }
 
+#if USE_ETHEREUM
+int ecdsa_get_ethereum_pubkeyhash(const uint8_t *pub_key, uint8_t *pubkeyhash)
+{
+	uint8_t h[65];
+	SHA3_CTX ctx;
+
+	if (!ecdsa_uncompress_pubkey(&secp256k1, pub_key, h)) {
```

Comment:
`ecdsa_uncompress_pubkey`, `ecdsa_read_pubkey`, `ecdsa_verify_digest`, `ecdsa_validate_pubkey` etc. all deal with public data. There is no need to clear stack variables. I suggest just removing MEMSET_BZERO calls from line 859 and 870 below.